db/schema.rb

```diff
@@ -26,6 +26,21 @@
     t.index ["walking_minutes_away"], name: "index_restaurants_on_walking_minutes_away"
   end
 
+  create_table "restaurants_tags", id: false, force: :cascade do |t|
```

Comment:
make sure your relations work in code since the table name is `restaurants_tags` but they `have_many :restaurant_tags`